cmd/gateway-azure.go

```diff
@@ -126,12 +126,18 @@ func azureToObjectError(err error, params ...string) error {
 
 // Inits azure blob storage client and returns AzureObjects.
 func newAzureLayer(args []string) (GatewayLayer, error) {
-	endPoint, secure, err := parseGatewayEndpoint(args[0])
-	if err != nil {
-		return nil, err
-	}
 
-	if endPoint == "" {
+	var endPoint string
+	var secure bool
+	var err error
+
+	if len(args) > 0 {
+		endPoint, secure, err = parseGatewayEndpoint(args[0])
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		// Provide default parameters if no arguments were provided
```

Comment:
Done,

I made another change which is enabling secure by default for Azure, does that look correct ?